packages/presentation/lib/app/app_bloc.dart

```diff
@@ -107,8 +110,18 @@ class _AppBloc extends BlocImpl implements AppBloc {
 
   @override
   void onSelectedTab(int index) {
-    if (selectedTab == index) return;
-    selectedTab = index;
+    final type = BottomNavBarItemTypeExtension.toType(index);
```

Comment:
u should call not extension, but enum here
```suggestion
    final type = BottomNavBarItemType.toType(index);
```